server/api/routes/reviewsRoutes.js

```diff
@@ -0,0 +1,42 @@
+const router = require('express').Router({ mergeParams: true });
+const { Sitter, Sitter_Review, Booking } = require('../../db');
+const { requireToken } = require('../authMiddleware');
+
+// get reviews of specific sitter (public)
+router.get('/:id', async (req, res, next) => {
```

Comment:
router.get('/:id/reviews', .... ) get all reviews where the sitterId on review === id from params
router.get('/:id/reviews/:reviewId') ....